ssn/rdf/examples/InkBird.ttl

```diff
@@ -0,0 +1,50 @@
+
+
+<https://example.org/sensor/IBS-TH2> rdfs:subClassOf sosa:Sensor ;
+    rdfs:comment "The class of IBS-TH2 sensors is a subclass of the general class of sensors" ;
+    gs1:pip <https://inkbird.com/products/hygrometer-ibs-th2> ;
+    rdfs:label "Bluetooth Temperature and Humidity Sensor IBS-TH2" ;
+    skos:notation "IBS-TH2" ;
+    sosa:observes <http://qudt.org/vocab/quantitykind/RelativeHumidity> ;
+    sosa:observes <http://qudt.org/vocab/quantitykind/Temperature> ;
+    system:hasSystemCapability [ 
```

Comment:
I don't  know `system` that well, but I doubt these object properties have owl:Class as range. Please verify.

Reply:
No, the range of `sosa:observes` is `sosa:Property`. 
http://qudt.org/vocab/quantitykind/Temperature is an individual of `rdf:type qudt:QuantityKind`, so it is inferred to also be `rdf:type sosa:Property` which is fine.

Reply:
In my understanding of OWL this is not fine. And a reasoner should fail to make that inference.

Perhaps it is best to verify this with a third person that knows OWL better.

Reply:
I added into this file
```
qk:RelativeHumidity 
    a sosa:Property ;
    a qudt:QuantityKind ;
. 
qk:Temperature 
    a sosa:Property ;
    a qudt:QuantityKind ;
. 
qudt:QuantityKind
    rdfs:subClassOf sosa:Property ;
.
```

OK?